Sources/Overload/OvTools/include/OvTools/Utils/PathParser.h

```diff
@@ -66,6 +66,12 @@ namespace OvTools::Utils
 		*/
 		static std::string GetExtension(const std::string& p_path);
 
+		/**
+		* Returns the file without the extension
+		* @param p_file
+		*/
+		static std::string GetFileWithoutExtension(const std::string& p_file);
```

Comment:
Might not be necessary, since this function would return `/path/to/foo` if given `/path/to/foo.txt`, which is fairly uncommon.

Alternatively, `std::filesystem::path::stem()` would return `foo`, and `std::filesystem::path::filename()` would return `foo.txt`.

So maybe in this use case, `stem()` would be the way to go